config/sync/field.storage.node.body.yml

```diff
@@ -1,9 +1,12 @@
+uuid: 2a6aa125-5aa1-4771-9438-090d04e2e00b
```

Comment:
due to this task, this file should not be overwritten:
https://app.scrumdo.com/projects/story_permalink/1437160